app/logic/events.py

```diff
@@ -253,6 +254,14 @@ def getUpcomingEventsForUser(user, asOf=datetime.datetime.now()):
                     .order_by(Event.startDate, Event.name).execute() # keeps the order of events the same when the dates are the same
                     )
 
+    bannedEvents = list(Event.select()
```

Comment:
I think we could benefit from putting this in its own logic function and testing separately as I can think of times in the future where this may be handy to have as a logic function.